tools/ldb_cmd.cc

```diff
@@ -498,9 +494,19 @@ bool LDBCommand::ParseStringOption(
 }
 
 Options LDBCommand::PrepareOptionsForOpenDB() {
-
-  Options opt = options_;
-  opt.create_if_missing = false;
+  ColumnFamilyOptions* cf_opts;
+  auto column_families_iter =
+      std::find_if(column_families_.begin(), column_families_.end(),
```

Comment:
where does `column_families_` come from?

Reply:
From the CFs in options file if provided; otherwise, by calling `DB::ListColumnFamilies`

Reply:
Notice if it comes from the CFs in options file, column_families_ will already be populated here so we modify its ColumnFamilyOptions directly. If it comes from `DB::ListColumnFamilies`, column_families_ will be populated after this function finishes, and its ColumnFamilyOptions will be created based on the Options object.

Reply:
thanks for answering.